set.mm

```diff
@@ -659770,7 +659702,7 @@ unification theorem (e.g., the sub-theorem whose assertion is step 5
     $( Equality theorem for restricted class abstractions.  (Contributed by
        Glauco Siliprandi, 26-Jun-2021.) $)
     rabeqd $p |- ( ph -> { x e. A | ch } = { x e. B | ch } ) $=
```

Comment:
This theorem can be removed!?

Reply:
Yes (same proof and same contributor). Since it has several usages, they should be adapted. @GinoGiotto : do you know how to do it easily ? (if you use metamath.exe, you can `minimize_with rabeqdv/may_grow` on all theorems using it, etc.; or you can save the proofs in the "normal" format, do the label change, and save the proofs back in the "compressed" format, I guess).

Reply:
Not necessary to save proofs in the normal format. I noticed replacing labels directly in the compressed proof works as well if the replaced statemetent is exactly identical to the old one (and when I finished I do `SAVE PROOF * /COMPRESSED /FAST`).

Reply:
Ah, it's true. I usually do the above trick when the theorems differ only by variable renamings, in which case you need to to the change on the uncompressed proof.